openhtf/io/test_record.py

```diff
@@ -71,3 +74,23 @@ class PhaseRecord(  # pylint: disable=too-few-public-methods,no-init
 
   See measurements.Record.GetValues() for more information.
   """
+
+
+class CodeInfo(mutablerecords.Record(
```

Comment:
i snuck that into #168 here:
https://github.com/google/openhtf/pull/168/files#diff-84ac26137df2222d79ee6e7afa7a3cc0R40
it may need some tweaks still to behave the way you guys want, but it's in there at least.